Source/Core/Common/IniFile.cpp

```diff
@@ -299,9 +299,13 @@ bool IniFile::Load(const std::string& filename, bool keep_current_data)
           // INI is a hack anyway.
           if ((key.empty() && value.empty()) ||
               (!line.empty() && (line[0] == '$' || line[0] == '+' || line[0] == '*')))
-            current_section->m_lines.push_back(line);
+          {
+            current_section->m_lines.push_back(std::string(line));
```

Comment:
Probably can be `emplace_back`, without constructing a `std::string` explicitly, too?